CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contribution Guide for dbt-ClickHouse Adapter
+
+## Introduction
+
+Thank you for considering contributing to the dbt-ClickHouse adapter! We value your contributions and appreciate your
+efforts to improve the adapter. This guide will help you get started with the contribution process.
+
+## Getting Started
+
+### 1. Fork the Repository
+
+Start by forking the repository on GitHub. This will create a copy of the repository under your own GitHub account.
+
+### 2. Set Up Environment
+
+* Make sure Python is installed locally.
```

Comment:
let's add a pointer to specific python version required

Reply:
Done